source/reference/method/MongoDBModelCollectionInfo-isView.txt

```diff
@@ -0,0 +1,53 @@
+========================================
+MongoDB\\Model\\CollectionInfo::isView()
+========================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Definition
+----------
+
+.. phpmethod:: MongoDB\Model\CollectionInfo::isView()
+
+   Return whether the collection is a :manual:`view </core/views/>`.
+
+   .. code-block:: php
+
+      function isView(): boolean
+
+Return Values
+-------------
+
+A boolean indicating whether the collection is a view.
+
+Examples
+--------
+
+.. code-block:: php
+
+   <?php
+
+   $info = new CollectionInfo([
+       'name' => 'foo',
+       'type' => 'view',
+       'info' => ['readOnly' => true],
+   ]);
```

Comment:
I'm aware this may be consistent with other examples for various `CollectionInfo` methods, but this isn't real-world usage of the class. Typically, we'd expect users to obtain `CollectionInfo` instances by calling `$database->listCollections()`. If we want to show a functional example, we should consider creating a view, then specifically printing the `CollectionInfo` instance for that view.